tests/test_modeldownload.py

```diff
@@ -23,6 +23,24 @@ def test_download_huggingface_model(tmp_path_factory, model="full_cat"):
     # Verify that the Hugging Face folder was removed
     assert not any(f.startswith("models--") for f in os.listdir(folder))
 
+def test_download_huggingface_superanimalmousemodel(tmp_path_factory, model="superanimal_topviewmouse"):
+    folder = tmp_path_factory.mktemp("temp")
+    dlclibrary.download_huggingface_model(model, str(folder))
+
+    assert os.path.exists(folder / "pose_cfg.yaml")
+    assert any(f.startswith("snapshot-") for f in os.listdir(folder))
+    # Verify that the Hugging Face folder was removed
+    assert not any(f.startswith("models--") for f in os.listdir(folder))
+
+def test_download_huggingface_superquadrupedmodel(tmp_path_factory, model="superanimal_quadruped"):
+    folder = tmp_path_factory.mktemp("temp")
+    dlclibrary.download_huggingface_model(model, str(folder))
+
+    assert os.path.exists(folder / "pose_cfg.yaml")
+    assert any(f.startswith("snapshot-") for f in os.listdir(folder))
+    # Verify that the Hugging Face folder was removed
+    assert not any(f.startswith("models--") for f in os.listdir(folder))
+
```

Comment:
These are tested with `download_all_models`?

Reply:
that's skipped.